src/components/TaskList.js

```diff
@@ -1,38 +1,45 @@
-import React from "react";
+import React from 'react';
 
-import Task from "./Task";
-import { isTag } from "postcss-selector-parser";
+import Task from './Task';
 
 const TaskList = props => {
-  console.log("ii", props.activeState);
+  console.log(props.todos);
+
+  let filteredContent = props.todos.filter(todo => {
```

Comment:
use `const` if the value is not changing.
```suggestion
  const filteredContent = props.todos.filter(todo => {
```